docs/getting-started/foundation/Foundation.stories.mdx

```diff
@@ -69,25 +69,25 @@ Once installed the default values of the CSS foundation can be customized to you
 
 ```css
 :root {
-    --o-ui-primary-50: #c63a0c;
-    --o-ui-apricot-50: #c63a0c;
-    --o-ui-overcast-50: #c63a0c;
-    --o-ui-desktop-50: #c63a0c;
+    --o-ui-alias-primary-50: #c63a0c;
+    --o-ui-global-apricot-50: #c63a0c;
+    --o-ui-global-overcast-50: #c63a0c;
+    --o-ui-global-desktop-50: #c63a0c;
```

Comment:
Is it still true that can redefine vars in `:root` ?

Should also remove "overcast"